main-installation.py

```diff
@@ -12,6 +12,12 @@ def on_error(error, items):
 
 analytics.write_key = 'jwq6QffjZextbffljhUjL5ODBcrIvsi5'
 
+integrations={
+  'Segment.io': {
+    apiHost: 'https://console.stage.redhat.com/connections/api',
```

Comment:
Is this meant to be a staging server or is this the production server name?

Reply:
This is the link that was shared with me by Heiko and Product management, I would assume this is correct